src/Volume.ts

```diff
@@ -27,7 +28,34 @@ export interface ImageInfo {
     translation: [number, number, number];
     rotation: [number, number, number];
   };
+  channelGui: ChannelGuiOptions[];
 }
+
+export const defaultImageInfo: ImageInfo = Object.freeze({
+  name: "",
+  version: "",
+  width: 1,
+  height: 1,
+  channels: 0,
+  tiles: 1,
+  pixel_size_x: 1,
+  pixel_size_y: 1,
+  pixel_size_z: 1,
+  channel_names: [],
+  channel_colors: [],
+  rows: 1,
+  cols: 1,
+  tile_width: 1,
+  tile_height: 1,
+  atlas_width: 1,
+  atlas_height: 1,
+  transform: {
+    translation: [0, 0, 0],
+    rotation: [0, 0, 0],
+  },
+  channelGui: [],
+});
```

Comment:
@toloudis came up with this default object

Reply:
The only thing that may be a bit risky is the 0 channels.  If something is broken (which it sounds like it isn't) we could try making one single channel in there.